python/feldera/sql_context.py

```diff
@@ -578,6 +578,71 @@ def start(self):
 
         self.__push_http_inputs()
 
+    def wait_for_idle(
+            self,
+            idle_interval_s: float = 5.0,
+            timeout_s: float = 600.0,
+            poll_interval_s: float = 0.2
+    ):
+        """
+        .. _wait_for_idle:
```

Comment:
We can remove this. 
If in the docs, we want to mention this function, we can just do: 
```
:meth:`.wait_for_idle`
```